src/zenml/config/compiler.py

```diff
@@ -346,58 +326,11 @@ def _compile_step(
         )
 
         complete_step_configuration = StepConfiguration(
-            docstring=step.__doc__, **step.configuration.dict()
```

Comment:
At what point are we now setting the step docstring?

Reply:
`BaseStep.__init__`